### PR TITLE
ztp: fix ArgoCD deployment policies-app-project.yaml path

### PR DESCRIPTION
**Pull request by mcornea**
Fix ArgoCD deployment kustomization.yaml to include the correct policies-app-project.yaml path.

### CODE REVIEW

**Review by serngawy — APPROVED** [review]
thanks for fixing that

File: ztp/source-crs/AcceleratorsCatalogSource.yaml [hunk 1]
```diff
@@ -0,0 +1,15 @@
+apiVersion: operators.coreos.com/v1alpha1
```
Comment (serngawy) [inline ztp/source-crs/AcceleratorsCatalogSource.yaml:1]: why adding this ? AND in which PGT (common, group, site) will be used ? There are already many CatalogSource exist under ztp/source-crs that can be overlay

Comment (mcornea) [reply ztp/source-crs/AcceleratorsCatalogSource.yaml]: Please ignore this, it was picked by mistake. I removed it.

**Review by serngawy — APPROVED** [review]

**Pull request merged**